modules/administration-guide/pages/configuring-routes.adoc

```diff
@@ -39,80 +35,7 @@ spec:
 <2> DNS name serviced by the target ingress controller
 <3> unstructured key value map stored with a resource
 
-* For a  new `{prod-cli}` installation:
-+
-. Configure the installation using:
-+
-[subs="+quotes,+attributes"]
-----
-$ {prod-cli} server:deploy --che-operator-cr-patch-yaml=patch.yaml ...
-----
-+
-The `patch.yaml` file must contain the following:
-+
-[source,yaml,subs="+quotes"]
-----
-spec:
-  server:
-    cheServerRoute:
-      labels: <labels> <1>
-      domain: <domain> <2>
-      annotations:     <3>
-        key1: value1
-        key2: value2
-    customCheProperties:
-      CHE_INFRA_OPENSHIFT_ROUTE_LABELS: <labels> <1>
-      CHE_INFRA_OPENSHIFT_ROUTE_HOST_DOMAIN__SUFFIX: <domain> <2>
-----
-<1> comma separated list of labels that are used by the target ingress controller to filter the set of Routes to service
-<2> DNS name serviced by the target ingress controller
-<3> unstructured key value map stored with a resource
 
-* For already existing {prod-short} installation:
-+
-. Update `{prod-checluster}` CR using the `{orch-cli}` tool:
-+
-.. To configure labels:
-+
-IMPORTANT: Use comma to separate labels: `key1=value1,key2=value2`
-+
-[subs="+quotes,+attributes"]
-----
-$ {orch-cli} patch checluster/{prod-checluster} -n {prod-namespace} --type=json -p \
-'[{"op": "replace", "path": "/spec/server/cheServerRoute/labels",'\
-'"value": "__<labels for a {prod-id-short} server route>__"}]'
-----
-+
-[subs="+quotes,+attributes"]
-----
-$ {orch-cli} patch checluster/{prod-checluster} -n {prod-namespace} --type=json -p \
-'[{"op": "replace", "path": "/spec/server/customCheProperties/CHE_INFRA_OPENSHIFT_ROUTE_LABELS", '\
-'"value": "__<labels for a workspace routes>__"}]'
-----
-+
-.. To configure domains:
-+
-[subs="+quotes,+attributes"]
-----
-$ {orch-cli} patch checluster/{prod-checluster} -n {prod-namespace} --type=json -p \
-'[{"op": "replace", "path": "/spec/server/cheServerRoute/domain",'\
-'"value": "__<ingress domain>__"}]'
-----
-+
-[subs="+quotes,+attributes"]
-----
-$ {orch-cli} patch checluster/{prod-checluster} -n {prod-namespace} --type=json -p \
-'[{"op": "replace", "path": "/spec/server/customCheProperties/CHE_INFRA_OPENSHIFT_ROUTE_HOST_DOMAIN______SUFFIX", '\
-'"value": "__<ingress domain>__"}]'
-----
-+
-.. To configure annotations:
-+
-IMPORTANT: Use object to specify annotations: `{"key1": "value1", "key2" : "value2"}`
-+
-[subs="+quotes,+attributes"]
-----
-$ {orch-cli} patch checluster/{prod-checluster} -n {prod-namespace} --type=json -p \
-'[{"op": "replace", "path": "/spec/server/cheServerRoute/annotations",'\
-'"value": __<annotations for a {prod-id-short} ingress>__}]'
-----
+.Additional resources
```

Comment:
I assume that the openshift-docs requirement to place an ID above the additional resources does not apply here.

Reply:
Yes